src/Azure.Functions.Cli/Common/Utilities.cs

```diff
@@ -263,5 +263,33 @@ internal static IConfigurationRoot BuildHostJsonConfigutation(ScriptApplicationH
             var configuration = builder.Build();
             return configuration;
         }
+
+        internal static bool IsMinifiedVersion()
+        {
+            var config = BuildArtifactsConfiguration();
+
+            try
+            {
+                var section = config.GetSection(Constants.MinifiedVersionConfigSectionName);
+
+                if (section.Exists())
+                {
+                    string value = section.Value;
+                    return bool.TryParse(value, out bool isValue) && isValue;
+                }
+            }
+            catch { }
+            return false;
+        }
+
+        internal static IConfigurationRoot BuildArtifactsConfiguration()
```

Comment:
Do we not have an existing `IConfiguration` to use? This seems overkill to use config builder to extract a single value from a json file (whose name is hard coded).

I am not familiar with what is all in `artifactsconfig.json` or its uses outside of this check are, but I would recommend the following:

1. Is it already loaded somewhere? Can we reuse that?
2. If not loaded, can we just use `System.Text.Json` to load _and cache_ that single value we need to a static variable?

Reply:
This class itself has configuration builder used a few times and its also there in [startup.cs](https://github.com/Azure/azure-functions-core-tools/blob/37b826f143b99cd62308b88320e71aa806f6197f/src/Azure.Functions.Cli/Actions/HostActions/Startup.cs#L97). Any suggestions on how can I combine/reuse them?

For now, `artifactsconfig.json` has only below content -

```json
{
    "minifiedVersion" : true
}
```

Reply:
Where does `artifactsconfig.json` come from?

Reply:
Its a new config file we are introducing with this PR - it may or may not be present in coretools. Any changes that should go to visual studio and not to cli/vscode can have this check - `IsMinifiedVersion() = true`. In BuildSteps.cs, we create minified version of coretools for Visual Studio case - [link](https://github.com/Azure/azure-functions-core-tools/blob/37b826f143b99cd62308b88320e71aa806f6197f/build/BuildSteps.cs#L121). We will need another PR to update this check in buildsteps.cs - probably create file `artifactsconfig.json` and make sure artifacts for both the scenarios vs and cli are being generated correctly.

Reply:
Oh, so this introduces that file. Let me think on this for a minute.